### PR TITLE
Docs-8390 SQS FIFO Queue

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

**Review by hreso110100 — DISMISSED** [review]

File: amazon-sqs/5.6/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc [hunk 1]
```diff
@@ -17,16 +17,17 @@ For more information about queue types, see https://docs.aws.amazon.com/AWSSimpl
...
 FIFO (First-In-First-Out) queues deliver messages in the order in which they are sent.
 
-To create the FIFO queue, use the *Create queue* operation in the connector, and set the *FifoQueue* and *ContentBasedDeduplication* field values to true.
+To create the FIFO queue, use the *Create queue* operation. When you configure this operation:
 
-End the *queue name* value with the `.fifo` extension: for example, `MyTestFIFOQueue.fifo`.
+* In the *Attributes* table, set the *FifoQueue* attribute to `true`.
+* Optionally, set the *ContentBasedDeduplication* attribute to `true` to have Amazon SQS generate the ID using the message body.
++
+The default for the *ContentBasedDeduplication* attribute is `false`. If you don't this field to `true`, users must set the *deduplicationId* attribute manually when sending messages to the queue, or the *Create queue* operation fails.
```
Comment (valkyrie69) [inline amazon-sqs/5.6/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc]:
```suggestion
The default for the *ContentBasedDeduplication* attribute is `false`. If you don't set this field to `true`, users must set the *deduplicationId* attribute manually when sending messages to the queue, or the *Create queue* operation fails.
```

File: amazon-sqs/5.6/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc [hunk 2]
```diff
@@ -17,16 +17,17 @@ For more information about queue types, see https://docs.aws.amazon.com/AWSSimpl
...
++
+* End the value for the *Queue name* field with the `.fifo` extension: for example, `MyTestFIFOQueue.fifo`.
```
Comment (valkyrie69) [inline amazon-sqs/5.6/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc]:
```suggestion
* End the value for the *Queue name* field with the `.fifo` extension, for example, `MyTestFIFOQueue.fifo`.
```

File: amazon-sqs/5.6/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc [hunk 3]
```diff
@@ -17,16 +17,17 @@ For more information about queue types, see https://docs.aws.amazon.com/AWSSimpl
...
+* Choose any region that Amazon SQS supports.
 
-While creating the FIFO queues, choose either of the two regions that Amazon supports:
-
-* US_EAST_2 (US East Ohio)
-* US_WEST_2 (US West Oregon)
-
-When using Amazon SQS Connector to send messages to the FIFO queue, use the same queue URL and region name that you used when creating the FIFO queue. You must also provide a value for the attribute `message group id` in the connector configuration while sending messages.
+When using Amazon SQS Connector to send messages to the FIFO queue, use the same queue URL and region name that Amazon SQS returned as the result of the *Create queue* operation. You must also provide a value for the attribute *message group id* field in the connector configuration while sending messages.
```
Comment (valkyrie69) [inline amazon-sqs/5.6/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc]:
```suggestion
When using Amazon SQS Connector to send messages to the FIFO queue, use the same queue URL and region name that Amazon SQS returned as the result of the *Create queue* operation. You must also provide a value for the attribute *Message group id* field in the connector configuration while sending messages.
```

Comment (valkyrie69) [reply amazon-sqs/5.6/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc]: I think this should be just Message group id field (no attribute in front of Message group id). Attribute and field are essentially the same thing so it's redundant to use both.

**Review by hreso110100 — DISMISSED** [review]

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**